src/containers/EventHandlers/market.ts

```diff
@@ -25,7 +25,7 @@ const MarketEvents = () => {
   useEffect(() => {
     const connectEvents = async () => {
       const api = await apiSingleton.loadInstance(network)
-      const grc20 = await grc20Singleton.loadInstance(api)
+      const VFT = await vftSingleton.loadInstance(api)
```

Comment:
IMO, it would be better to stick with the convention of naming instances using lowercase, such as `vft`